plugins/octant/cmd/antrea-octant-plugin/traceflow.go

```diff
@@ -313,6 +316,19 @@ func (p *antreaOctantPlugin) actionHandler(request *service.ActionRequest) error
 			return nil
 		}
 		return nil
+	case runTraceAgainAction:
+		// Check if traceflow has been run before
+		if mostRecentTraceflow == nil {
+			log.Printf("Failed to run traceflow again: Run a traceflow before attempting to run a traceflow again.\n")
```

Comment:
the log.Printf will only add a log in backend only, it's better to return an alert to UI as well so user can learn what's to do first. eg:
```
alert := action.CreateAlert(action.AlertTypeError, "Failed to run traceflow again: Use 'START NEW TRACE' or 'START NEW LIVE-TRAFFIC TRACE' to run a traceflow before attempting to run a traceflow again.", action.DefaultAlertExpiration)
request.DashboardClient.SendAlert(request.Context(), request.ClientID, alert)
```

Reply:
Sounds good, I'll add this.